code-challenges.js

```diff
@@ -59,3 +129,22 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+const minusNumbers = (nums1,nums2) => {
+    if (nums1 > nums2){
+        console.log(nums1 - nums2)
+    } else if (nums2 > nums1){
+        console.log(nums2 - nums1)
+    }
+    else{
+
+    }
+}
+
+ console.log(minusNumbers(100,58))
+
+//The function produces the answer perfectly, but why is there an undefined message below the my answer? Is there a seperate console.log somewhere else in this editor?
+//It seems like I am recalling a varible in this problem first without giving it a value...
+//seems like the way im writing my console.log is incorrect?
+//I've tested the code in the googlechrome console and still receive the undefined message, the debugger told be the undefined statement came from my console.log command
+
```

Comment:
If you call a function without a return statement inside the function, you will receive `undefined`. JavaScript follows the following rule: functions always return something. We use the keyword `return` tell the function to stop and return a value. Any code located after a return statement will never be executed. 
So if you were to place `return` before the `console.log `that is within your function, you will receive a value and then an undefined message. The undefined message is for the console.log because the purpose of console.log is printing stuff to the console. This console.log itself cannot be executed after the return statement. 
However, the console.log outside the function or before a return statement will work. I suggest testing it out and seeing it for yourself.